eva/executor/load_multimedia_executor.py

```diff
@@ -41,14 +43,27 @@ def exec(self):
         table_obj = None
         try:
             valid_files = []
-            for file_path in iter_path_regex(self.node.file_path):
+            search_path = self.node.file_path
+
+            # If it is a s3 path, download the file to local
+            if search_path.as_posix().startswith("s3:/"):
+                s3_dir = Path(
+                    ConfigurationManager().get_value("storage", "s3_download_dir")
+                )
+                dst_path = s3_dir / self.node.table_info.table_name
+                dst_path.mkdir(parents=True, exist_ok=True)
+                download_from_s3(self.node.file_path, dst_path)
+                search_path = dst_path / "*"
```

Comment:
If we fire multiple load commands on the same table, will it end up loading the same videos multiple times (because we download them in the same directory)? How about we not do this and rather iterate over the downloaded files?